### DISCUSSION TITLE
Finding gpc

### DISCUSSION THREAD

**Discussion by progfrosty** [samiam95124/Pascal-P6#120] [General]
Would love to give P6 a go but I've struggled to find a gpc I can install to get started.

I wonder if its possible to use fpc at all ?

**Comment by samiam95124** [comment]
progfrosty,

Please see 14.3.1.2 of the manual, page 67. See also the discussion at

https://github.com/samiam95124/Pascal-P6/discussions/51

I only use GPC with Ubuntu 20.*, I find that even other linuxs fail to work with GPC.

I have not had luck using FPC to compile Pascal-P5, so I assume that the same issues
would exist with Pascal-P6. It would need to be modified to work with FPC.

This is the reason I am working to have Pascal-P6 self compile.

Regards,

Scott Franco

**Comment by samiam95124** [comment]
See recent post on fpc.
